pkg/apis/apps/v1alpha1/servicebinding_types.go

```diff
@@ -0,0 +1,108 @@
+package v1alpha1
+
+import (
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+
+// ServiceBindingSpec defines the desired state of ServiceBindingSpec
+type ServiceBindingSpec struct {
+	// MountPathPrefix is the prefix for volume mount
+	// +optional
+	MountPathPrefix string `json:"mountPathPrefix,omitempty"`
+
+	// EnvVarPrefix is the prefix for environment variables
+	// +optional
+	EnvVarPrefix string `json:"envVarPrefix,omitempty"`
+
+	// DataMapping is used to map CR/CSV/CRD attributes to Custom env variables
+	// +optional
+	DataMapping []corev1.EnvVar `json:"dataMapping,omitempty"`
```

Comment:
Since `DataMapping` is a list, should it be plural (`DataMappings`)?

Reply:
Shall we revisit this rename? 

The reason for this is is the input is a map of environment variable names and templates and its output are rendered environment variables, which makes it much closer to `customEnvVars` than `dataMapping`.

Reply:
Sure! I don't mind revisiting the name